src/lms-ds-loader/edfi_lms_ds_loader/migrator.py

```diff
@@ -23,6 +23,7 @@
     "create_user_tables",
     "create_section_tables",
     "create_assignment_tables",
+    "create_section_association_tables",
```

Comment:
You can put it after create_processed_files_table 🤔

Reply:
Yes, it is best to add the migration at the end of the list. But probably doesn't really matter in this case.

Reply:
Given the stage of development we're at, I wasn't concerned beyond making sure relational dependencies are in order.  I can't imagine not dropping/recreating dbs from scratch at this point during testing.

Reply:
Fair enough